docs/BUILDING_DOCKER.md

```diff
@@ -38,10 +38,4 @@ docker run -it --rm --mount type=bind,source="$(pwd)",destination=/mm mm "/usr/b
 
 ## 4. Setup and Build the ROM
 
-Once inside the container, you will need to set the repository as a trusted repository with the following command:
-
-```bash
-git config --global --add safe.directory /mm
```

Comment:
The `Dockerfile` does this already, so I thought it was redundant, correct me if I'm wrong because I'm no Docker expert.